src/App.jsx

```diff
@@ -5,18 +5,19 @@ import axios from 'axios';
 
 import RecipeBrief from './components/RecipesDisplay/RecipeBrief';
 import Header from './components/header/Header';
+import SearchBar from './components/searchBar/SearchBar';
 
 function App() {
```

Comment:
You should reserve the App file for more providers and context related wrappers. And wrap them around a main page file that should hold all the logic you have currently on this page.